gui/objects_map/main_names.py

```diff
@@ -22,3 +22,8 @@
 
 # Banners
 secureYourSeedPhraseBanner_ModuleWarning = {"container": statusDesktop_mainWindow, "objectName": "secureYourSeedPhraseBanner", "type": "ModuleWarning", "visible": True}
+
+# Notifications
```

Comment:
For toast messages we already have a separate class (https://github.com/status-im/desktop-qa-automation/blob/master/gui/components/wallet/wallet_toast_message.py). I made some changes to this class in my PR here https://github.com/status-im/desktop-qa-automation/pull/336, so after it's merged we can use it in your PR too. 

Object names for toast messages are already here - https://github.com/status-im/desktop-qa-automation/blob/master/gui/objects_map/component_names.py#L284

Reply:
fixed